src/main/java/com/bookstore/bookstoreapi/data/entity/Customer.java

```diff
@@ -41,6 +41,11 @@ public class Customer {
   @Column(name = "Address", nullable = false)
   private String address;
 
+  @Column(name = "Ratings", nullable = true)
+  @OneToMany
+  @Builder.Default
+  private Set<Rating> ratings = new HashSet<>();
```

Comment:
As I said before, this is the minor owner and the `Rating` entity class is the actual owner., you can call it, which means it needs to reverse the relationship, please refer to my code to understand how is that done.